crates/uv/tests/it/python_install.rs

```diff
@@ -1370,3 +1371,518 @@ fn python_install_cached() {
       Caused by: An offline Python installation was requested, but cpython-3.12.10[DATE]-[PLATFORM].tar.gz) is missing in python-cache
     ");
 }
+
+// A virtual environment should track the latest patch version installed.
+#[test]
```

Comment:
Any tests that use `python install` will need to be under the `python-managed` flag

Reply:
All tests in this module (and my new [`python_upgrade.rs` module](crates/uv/tests/it/main.rs)) are under the `python-managed` flag, unless I'm missing something.

Reply:
Ah okay I forgot it was at the module level 👍 thanks!